pkg/config/paths.go

```diff
@@ -0,0 +1,46 @@
+package config
+
+import (
+	"fmt"
+	URL "net/url"
+	"os"
+	"path"
+	"strings"
+
+	"github.com/PuerkitoBio/purell"
+)
+
+func GetBase(base *string) string {
+	if base == nil || *base == "" {
+		baseUrl, err := os.Getwd()
+		if err != nil {
+			return ""
+		}
+		return baseUrl
+	} else {
+		return *base
+	}
+}
+
+func NormalizeUrl(url string, base ...string) (string, error) {
+	var normalUrl *URL.URL
+	var err error
+	var baseUrl string
+	if len(base) > 0 {
+		baseUrl = GetBase(&base[0])
+	} else {
+		baseUrl = GetBase(nil)
+	}
+	if strings.HasPrefix(url, ".") {
```

Comment:
If `url` contains `..`, is it possible to exploit the FS? Just something to think about.